src/Layout/Navigation/RecentList.tsx

```diff
@@ -15,31 +15,13 @@ import {
   NavGroup,
   NavList,
 } from '@patternfly/react-core';
-import { PlusIcon } from '@patternfly/react-icons';
 
 import NavigationRecentItem from './RecentItem';
-import { NavigationItemObject, NavigationRecentItemObject } from '.';
-import NavigationMainItem from './MainItem';
+import { NavigationRecentItemObject } from '.';
 import { ROUTE } from '../../route.enum';
 
-import styles from './index.module.css';
-import { buildGettingStartedPath } from '../../services/helpers/location';
 import { History } from 'history';
 
-function buildCreateWorkspaceItem(): React.ReactElement {
-  const createWorkspacePath = buildGettingStartedPath('custom-workspace');
-  const item: NavigationItemObject = {
-    to: createWorkspacePath,
-    label: 'Create Workspace',
-    icon: <PlusIcon className={styles.mainItemIcon} />,
```

Comment:
please clean up this rule from the corresponding CSS module.

Reply:
I will do it